GitHub-Test-4-PyNLPDiscuss/scatledgeGitTest4.md

```diff
@@ -0,0 +1,9 @@
+# Git Test Part 4
+
+## I didn't do the discussion post so i will make upsome stuff
+
```

Comment:
While I see that you did not participate in the discussion I will still comment here so I can meet the requirements for GitTest5! Instead I'll comment on the different md files that you used! cool idea using italics! I decided to include another code block in mine! I also included a link in mine due to my original discussion post already containing one!.